__docs__/foundations-color.mdx

```diff
@@ -285,14 +285,22 @@ backgrounds in light mode.
 
 <ColorGroup colors={semanticColor.text} group="text" />
 
-### Border
+### Core
```

Comment:
Would it be helpful to include docs/guidelines around when core tokens should be used? vs the other semantic color tokens!

Reply:
Yes totally, but I'm not fully sure how to document that as we haven't converted things to use `core`. Would it be ok if we write this document after migrating from `action` to `core` tokens in `webapp`? if that makes sense ofc.